<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="shadowframe">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(56, 58, 89);
	color:rgb(220, 220, 220);
	border-radius:20px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="selectfile">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>401</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(56, 58, 89);
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Select file (.pptx)</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="loadingBar">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{
	background-color: rgb(195, 195, 195);
	
	color: rgba(255, 255, 255, 0);
	border-style:none;
	border-radius:10px;
	text-align:center;
}
QProgressBar::chunk{
	border-radius:10px;
	background-color: rgb(255, 255, 255);
	
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(56, 58, 89, 0);
color: rgb(56, 58, 89);</string>
       </property>
       <property name="text">
        <string>Processing file...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(56, 58, 89,0);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
image: url(:/newPrefix/Icons/settings.png);</string>
       </property>
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(56, 58, 89,0);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>close</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="start_button">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Start Presenting</string>
       </property>
      </widget>
      <widget class="QLabel" name="start_tag">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Note:&lt;/strong&gt; Please open the powerpoint app before start presenting</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
